### DISCUSSION TITLE
Allow associating comments with card data

### DISCUSSION THREAD

**Discussion by megabug** [TeamWalrus/tusk#48] [Ideas]
You should be able to edit comments in the read card data view.

In addition, once #26 is implemented, you should be able to start card reading with a default comment for all cards read in that session.

**Comment by dunderhay** [comment]
This will start adding a bit of complexity because there are no ID's for card data entries. Data isn't stored in a database, just in a large card dump file (cards.jsonl), so looking up an entry to add a comment will be a hassle. Thoughts on how to do this?
